juego.py

```diff
@@ -37,21 +37,47 @@ def __init__(self, level, fuente):
         self.expresion = fuente.render(self.primero + simbolo[operador] + self.segundo + " = ? ",True,(255,0,0))
         self.resultado = str(eval(self.primero+operacion[operador]+self.segundo))
         self.vida = 0
-                                           
+        
+        no_repeat_check_x = []
+        no_repeat_check_y = []
+        list_y = [int(sx(-50)),int(sx(-40)),int(sx(-30)),int(sx(-10)),int(sx(-20)),int(sx(-0)),int(sx(-60)),int(sx(-65))]
+        list_x = [int(sx(100)),int(sx(200)),int(sx(300)),int(sx(400)),int(sx(500)),int(sx(600)),int(sx(700)),
+                int(sx(800)),int(sx(900))]
```

Comment:
Is there any reason why `list_x` has more values than `list_y`?

Reply:
> Is there any reason why `list_x` has more values than `list_y`?

Thanks for reviewing. There is no pressing reason as such. Since the x and y coordinates are chosen randomly and independent of each other, availing only 6 coordinates would serve the purpose. My intention is to cover the entire screen, hence the x- coordinates ranges from 100 to 900. The purpose of slightly varying the y coordinates is to  prevent the overlapping, if the multiple 4 digit number appears consecutively.The difference in the number of values wouldn't be of much concern to the output. However, I will consider adding one more value to ```list_y``` which  might as well increase a possibility.